### PR TITLE
Fix executable path in ffmaster for Linux.

### PR DESCRIPTION
**Pull request by mzf-guest**
Update the executable path In ffmaster for Linux.

Thanks,
François

### CODE REVIEW

File: plugin/seq/ffmaster.c [hunk 1]
```diff
@@ -95,7 +95,7 @@ int main(int argc, const char **argv) {
                 "..\\examples\\plugin\\ffslave.edp",
                 "-nw", "-ns", "-v", ff, NULL );
 #else
-  sprintf(ff, "FreeFem++.exe C:\\Users\\t1m\\FreeFem++\\examples\\plugin\\ffslave.edp -nw -ns -v %d&", debug);
+  sprintf(ff, "FreeFem++ /usr/share/doc/freefem++/examples/plugin/ffslave.edp -nw -ns -v %d&", debug);
```
Comment (prj-) [inline plugin/seq/ffmaster.c:98]: Shouldn't this be a relative path like on Windows, i.e.,
```suggestion
  sprintf(ff, "FreeFem++ ../examples/plugin/ffslave.edp -nw -ns -v %d&", debug);
```

Comment (prj-) [reply plugin/seq/ffmaster.c:98]: Just FYI, I've tested my suggested change and it's running OK. So if it's OK for you as well, I'd prefer that you apply my suggestion.

Comment (mzf-guest) [reply plugin/seq/ffmaster.c:98]: Hello,

unfortunately the relative path won't work on Debian and derivatives where example files are installed under `/usr/share/doc` and binaries under `/usr/bin` with no guarantees of the current working directory.

We could imagine either a check if the file exists at runtime by testing several path combinations, or setting the path at compile time in the configuration phase.

What do you think?

Thanks,

François

Comment (prj-) [reply plugin/seq/ffmaster.c:98]: OK, I don't care about that file. Clearly, it was broken before. It will still be broken, but that's no big deal.

**Pull request merged**